sdk/tables/azure-data-tables/inc/azure/data/tables/account_sas_builder.hpp

```diff
@@ -122,55 +105,20 @@ namespace Azure { namespace Data { namespace Tables { namespace Sas {
      */
     Delete = 4,
 
-    /**
-     * @brief Indicates that deleting previous blob version is permitted.
-     */
-    DeleteVersion = 8,
-
     /**
      * @brief Indicates that List is permitted.
      */
-    List = 16,
+    List = 8,

```

Comment:
Given that you don't reassign List from 8 to 1, do you want to keep "Add" and "Update" with their old numbers, so they are, for whatever reason compatible?